public/javascripts/livevalidation_standalone.js

```diff
@@ -314,17 +314,18 @@ LiveValidation.prototype = {
      */
     validate: function(){
       if(!this.element.disabled){
-		var isValid = this.doValidations();
-		if(isValid){
-			this.onValid();
-			return true;
-		}else {
-			this.onInvalid();
-			return false;
-		}
-	  }else{
-      return true;
-    }
+        var isValid = this.doValidations();
+        if(isValid){
```

Comment:
As long as this line is being edited anyway, can you update the spacing on this line to `if (isValid) {`?